guides/v2.3/config-guide/elasticsearch/es-config-stopwords.md

```diff
@@ -126,3 +126,22 @@ To change the directory:
     ```
 
 Save your changes to `di.xml` and exit the text editor.
+
+To change the directory from your module
+
+1. [Create a new module](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/build/module-file-structure.html){:target="_blank"}
+
+2.	In your module `etc/di.xml` add instructions:
+
+```xml
+
+<type name="Magento\Elasticsearch\SearchAdapter\Query\Preprocessor\Stopwords">
+	<arguments>
+		<argument name="stopwordsModule" xsi:type="string">(Your_Module)</argument>
+		<argument name="stopwordsDirectory" xsi:type="string">stopwords</argument>
+	</arguments>
+</type>
+```
+3. In your module, create the directory `etc/stopwords`, inside add files .csv
```

Comment:
Please, change the order of the words in the following explanation. 
```suggestion
3. In your module, create the directory `etc/stopwords`, with the corresponding `.csv` files inside.
```

Also, it would be really nice to have a reference (link) to the existing stopwords implementation in the `Magento_Elasticsearch` module.